sections/tooling/babel-plugin.js

```diff
@@ -16,6 +16,8 @@ const BabelPlugin = () => md`
 
   Then add it to your babel configuration like so:
 
+  > The plugin call order in your .babelrc file matters. You should put the babel-plugin into your env's plugin. See [this](https://github.com/styled-components/babel-plugin-styled-components/issues/78) for more information. ⚠️
```

Comment:
> You should put the babel-plugin into your env's plugin.

Can we clarify this a little please?

I.e. "If you're using the `env` property in your babel configuration, then putting this plugin into the `plugins` array won't suffice. Instead it needs to be put into each env's `plugins` array to maintain it being executed first."

Also can you wrap ".babelrc" in backticks please?